### PR TITLE
Allow pandas.DataFrame data types

### PR DESCRIPTION
**Pull request by terrytangyuan**
In the future, we may want to allow wider range of data types but this one is very common and should be a good start.

### CODE REVIEW

File: skflow/__init__.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 from skflow.trainer import TensorFlowTrainer
 from skflow import models, data_feeder
 from skflow import preprocessing
-
+from skflow.io.pandas_io import *
```
Comment (ilblackdragon) [inline skflow/__init__.py]: what about just `from skflow.io import *` ? also do we want to just drop everything in skflow namespace? maybe instead `from skflow import io`?

Comment (terrytangyuan) [inline skflow/__init__.py]: I am changing it to `from skflow.io import *`, feel free to change it later to something else.

File: skflow/__init__.py [hunk 2]
```diff
@@ -67,6 +67,13 @@ def __init__(self, model_fn, n_classes, tf_master="", batch_size=32, steps=50, o
         self.num_cores = num_cores
         self._initialized = False
 
+    def _data_type_filter(self, X, y):
```
Comment (terrytangyuan) [inline skflow/__init__.py]: Do you think this method should be in `io.pandas_io.py`?

Comment (ilblackdragon) [inline skflow/__init__.py]: I think it makes sense somewhere around data feeders, but they are not used right now for predict... We can keep it here for now and then move as more variety of io stuff will be needed.

**Comment by terrytangyuan** [comment]
@ilblackdragon Fixed!

**Pull request merged**